consensus/src/chained_bft/persistent_liveness_storage.rs

```diff
@@ -61,6 +63,16 @@ pub struct LedgerRecoveryData<T> {
     genesis_root: Option<RootInfo<T>>,
 }
 
+impl<T: Payload> fmt::Debug for LedgerRecoveryData<T> {
```

Comment:
we can't derive Debug?

Reply:
Yeah we can't, some of the field in `LedgerRecoveryData` doesn't impl Debug trait

Reply:
just derive Debug on all data?

Reply:
`ValidatorVerifier` uses `BTreeMap` which is in std library but doesn't impl Debug

Reply:
ah, ok, but you don't need this Debug since we're moving away from using Result right?